app/src/main/AndroidManifest.xml

```diff
@@ -21,6 +21,7 @@
         android:roundIcon="@drawable/app_icon_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+        <activity android:name=".activity.CompassActivity"></activity>
```

Comment:
Change 
`<activity android:name=".activity.CompassActivity"></activity>`
to
`<activity android:name=".activity.CompassActivity"/>`